### ISSUE TITLE
When using the auto configuration for lets encrypt, do the certs auto renew?

### ISSUE THREAD

**Issue by runandrew** [virtkick/http-master#129]
When using the auto configuration for lets encrypt, do the certs auto renew? I tried to look in the documentation or any other issues, and could not find information on this topic.

**Comment by Nowaker** [comment]
Yes, http-master is supposed to auto-renew them. And yes, there appears to be a bug - same problem happened to me very recently.

**Comment by runandrew** [comment]
I haven't had my certs long enough to go through an auto-renew, so I don't know if they renew correctly or not. I'm just planning ahead so there is no downtime.

@Nowaker  So you had a problem with it not renewing? Should I just configure the SSL manually?
